### ISSUE TITLE
Plant Harvest Yield does not increase amount harvested

### ISSUE THREAD

**Issue by RelaxedOtter** [Alexis-Grimm/Un-Limited#2]
Thanks for this mod, I loved having it.
Doing a bit of testing (as I'm making some new inspirations), the patch for plant harvest yield does not change the amount harvested (in 1.0 it definitely did - not sure whether it was this mod or the [SYR] Harvest Yield Patch mod that removed the limit as I had bothrunning back then). Additional testing with mining yield shows that that it working completely as expected (yielding more or less steel from mining depending on yield level).

7 different pawns, all with identical stats and different implants to modify the yield amount (all with skill 3 - so a baseline of 80% harvest yield) - on Cassandra Rough harvesting Corn:
 
- Pawn 1: Display 100%, Harvested amount: 22
- Pawn 2: Display: 190%, Harvested amount: 22
- Pawn 3: Display: 110%, Harvested amount: 22
- Pawn 4: Display: 115%, Harvested amount: 22
- Pawn 5: Display: 126%, Harvested amount: 22
- Pawn 6: Display: 275%, Harvested amount: 22
- Pawn 7: Display: 88%, Harvested amount: 22

Compare that to the mining yield stats (skill 8 so baseline of 100% mining yield)
- Pawn 1: Display 100%, Mined amount: 40
- Pawn 2: Display 190%, Mined amount: 76
- Pawn 3: Display 110%, Mined amount: 44
- Pawn 4: Display 115%, Mined amount: 46
- Pawn 5: Display 126%, Mined amount 50
- Pawn 6: Display 275%, Mined amount 110
- Pawn 7: Display 88%, Mined amount: 36
So mining yield is working 100% as expected. 

Using an inspiration to directly buff the mining yield stat by 75% correctly increased yield by that amount. Using one to directly buff the plant harvest yield stat by 75% did nothing - still the same 22 harvest.

Mods in use were: Core, Royalty, EdB Prepare Carefully, Un-Limited, Archotech Expanded (this was used for the Archotech Brain Melder on Pawn 6 as it gives +300 consciousness which was giving massive harvest yield amounts in 1.0)

**Comment by RelaxedOtter** [comment]
Looking further into the [SYR] Plant Harvest Yield Patch, that is actually a small Harmony code patch that changes the functionality of this stat to work the same way at the mining yield stat and not be a chance of harvesting success. So not an issue for this mod, just a really poorly named stat on Rimworld's part. Closing.

**Comment by Alexis-Grimm** [comment]
Yes, I've known about this for a while, I've always suggested the use of that mod for anyone that wanted that. I doubt that I will add a patch for this myself, however I will think about it, and keep this open until I decide

**Comment by sza95** [comment]
Please think about it, I'm using that mod and it also removes the limit of butchery, mining and dissembling but if you load it after your mod it'll throw errors, I guess the mod doesn't check if there's any other mod doing this as well so I have to load yours after that one also it's better to have everything in one single mod than to have 2 or 3 to archive what you need/want.

**Comment by Alexis-Grimm** [comment]
I don't expect to do this, however with the most recent update (1.2.2) I have included a manifest and such. When using fluffy's or rimpy mod manager it should specify that this mod should be loaded after plant harvest patch to avoid errors. As far as I know they still work fine together otherwise. I still need to do research on the new vanilla mod manager load order stuff, because I tried to set it up there as well, and failed.
